GRDB/Core/DatabasePool.swift

```diff
@@ -206,16 +208,15 @@ extension DatabasePool {
     }
     
     
-    #if os(iOS)
+    #if canImport(UIKit)
     /// Listens to UIApplicationDidEnterBackgroundNotification and
     /// UIApplicationDidReceiveMemoryWarningNotification in order to release
     /// as much memory as possible.
     ///
     /// - param application: The UIApplication that will start a background
     ///   task to let the database pool release its memory when the application
     ///   enters background.
-    public func setupMemoryManagement(in application: UIApplication) {
-        self.application = application
+    private func setupMemoryManagement() {
```

Comment:
Keep the setupMemoryManagement() method public, but deprecated with message "Memory management is now enabled by default. This method does nothing."